ddp-workspace/projects/ddp-mbc/src/assets/i18n/es.json

```diff
@@ -47,79 +47,77 @@
     },
     "Toolkit": {
         "Common": {
-            "Organization": "[es]Count Me In"
+            "Organization": "Cuenten Conmigo"
         },
         "Header": {
-            "HeaderLogo": "[es]Metastatic<br>Breast Cancer<br>Project",
-            "DataRelease": "[es]Data Release",
-            "LearnMore": "[es]Learn More",
-            "CountMeIn": "[es]count me <span class=\"icon-silhouette\"></span>n"
+            "HeaderLogo": "Metastatic<br>Breast Cancer<br>Project",
+            "DataRelease": "Datos",
+            "LearnMore": "Aprenda Más",
+            "CountMeIn": "cuenten conmigo"
```

Comment:
Did you skip the icon (`<span class=\"icon-silhouette\"></span>`) on purpose ?
We can do similar approach in Spanish:

![image](https://user-images.githubusercontent.com/7396837/115040691-5caf8d00-9eda-11eb-94ae-927e56359eee.png)

![image](https://user-images.githubusercontent.com/7396837/115040846-88cb0e00-9eda-11eb-8ed2-62e931c73c5f.png)

Reply:
Told @aglushko-broad that almost nobody will notice the little person, but someone will and this will make them happy!

Reply:
done!